lib/kitchen_sink/map.ex

```diff
@@ -56,7 +56,9 @@ defmodule KitchenSink.Map do
     right = clean_struct(right)
     do_deep_merge(left, right)
   end
-
+  # When the left argument is a struct, ensure that its own keys are the only
```

Comment:
The comment is good, but add a space between this and the end of the previous function.